apis/python/interactor.py

```diff
@@ -1,43 +0,0 @@
-#!/usr/bin/env python -i
```

Comment:
I haven't used this script in ages (a week, if not two) -- if anyone else uses it lmk, else I plan to jettison it.